docs/Maths.md

```diff
@@ -226,106 +225,16 @@ Result:
-w = 100
-h = 70
-
-# Make the PDF
-pdf = FPDF()
-pdf.add_page()
-pdf.image(svg_img_byte, x=x, y=y, w=w, h=h)
-pdf.output('bar_chart.pdf')
-```
-The above code generates a pdf with title `bar_chart.pdf` file with following graph -
-![](pygal_chart_cairo.PNG)
-
-**!! Troubleshoot: !!**
-
-You may encounter `GTK` (Gnome Toolkit) errors while executing the above example in windows. Error could be like following -
-```
-OSError: no library called "cairo-2" was found
-no library called "cairo" was found
-no library called "libcairo-2" was found
-cannot load library 'libcairo.so.2': error 0x7e
-cannot load library 'libcairo.2.dylib': error 0x7e
-cannot load library 'libcairo-2.dll': error 0x7e
-```
-In this case install install `GTK` from [GTK-for-Windows-Runtime-Environment-Installer](https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases). Restart your editor. And you are all done.
-
-### Using svglib and reportlab (*A slower and purely pythonic implementation*) ### 
-```python
-import io
-import pygal
-from reportlab.graphics import renderPM
-from svglib.svglib import SvgRenderer
-from fpdf import FPDF
-from lxml import etree
-
-# Create a Pygal bar chart
-bar_chart = pygal.Bar()
-bar_chart.title = 'Sales by Year'
-bar_chart.x_labels = ['2016', '2017', '2018', '2019', '2020']
-bar_chart.add('Product A', [500, 750, 1000, 1250, 1500])
-bar_chart.add('Product B', [750, 1000, 1250, 1500, 1750])
-
-# Render the chart and convert it to a bytestring object
-svg_img = bar_chart.render()
-svg_root = etree.fromstring(svg_img)
-drawing = SvgRenderer(svg_img).render(svg_root)
-drawing_img_byte = renderPM.drawToString(drawing)
-img_bytes = io.BytesIO(drawing_img_byte)
-
-# Set the position and size of the image in the PDF
-x = 50
-y = 50
-w = 100
-h = 70
-
-# Make the PDF
-pdf = FPDF()
-pdf.add_page()
-pdf.image(img_bytes, x=x, y=y, w=w, h=h)
-pdf.output('bar_chart_pdf.pdf')
-```
-User who are using `reportlab` and `svglib` to work with `svg` images and are intended to render `svg` images `reportlab` and `svglib` can use this library. However, it consumes a little bit more time than the previous example.
+### Using Pygal ###
+[Pygal](https://www.pygal.org/en/stable/) is a Python graph plotting library.
+You can install it using: `pip install pygal`
 
-The above code shows following output -
-![](pygal_chart.png)
+`fpdf2` can embed graphs and charts generated using `Pygal` library. However, they cannot be embedded as SVG directly, because `Pygal` inserts `<style>` & `<script>` tags in the images it produces (_cf._ [`pygal/svg.py`](https://github.com/Kozea/pygal/blob/3.0.0/pygal/svg.py#L449)), which is currently not supported by `fpdf2`.
+The full list of supported & unsupported SVG features can be found there: [SVG page](SVG.md#supported-svg-features).
 
-**Why there is a performance issue between `cairosvg` and `svglib`?**
+You can find documentation on how to convert Pygal vector images (SVG) to raster images (PNG, JPG) there:
```

Comment:
As this section portrays on embedding `Charts` and `Graphs` on `fpdf` generated by `Pygal` this line can be rephrased as following -
Find the documentation on generating `Charts` and `Graphs` using `PyGal` and embedding them into `fpdf` in [Converting vector graphics to raster graphics](SVG.md#converting-vector-graphics-to-raster-graphics). 

The rest of the things looks good to me.

Reply:
Thank you for the review @ssavi-ict! 😊

You are right, this sentence could have been improved, I have just made an edit.

Merging now

Reply:
My Pleasure